### PR TITLE
Update Swoole ini

### PR DESCRIPTION
**Pull request by sy-records**

### CODE REVIEW

File: reference/swoole/ini.xml [hunk 1]
```diff
@@ -47,6 +47,30 @@
       <entry>PHP_INI_SYSTEM</entry>
       <entry><!-- leave empty, this will be filled by an automatic script --></entry>
      </row>
+     <row>
+      <entry><link linkend="ini.swoole.enable-coroutine">swoole.enable_coroutine</link></entry>
+      <entry>On</entry>
+      <entry>PHP_INI_ALL</entry>
+      <entry><!-- leave empty, this will be filled by an automatic script --></entry>
```
Comment (cmb69) [inline reference/swoole/ini.xml:54]: Unfortunately, that's not true. There is no automatic script which fills the changelog column. Could you please manually add availability information? I mean something like: `Available as of swoole x.y.z` (of course, replacing x.y.z with the version number where the INI setting has been added).

**Review by cmb69 — CHANGES_REQUESTED** [review]
Thank you very much! Please address my comment. :)

File: reference/swoole/ini.xml (hunk 1 above)
Comment (sy-records) [reply reference/swoole/ini.xml:54]: Sure, I'll add it later.

Comment (sy-records) [reply reference/swoole/ini.xml]: done.

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request closed without merging**